mrmustard/lab/gates.py

```diff
@@ -97,8 +96,9 @@ def U(self, cutoffs: Sequence[int]):
         """Returns the unitary representation of the Displacement gate using the Laguerre
         polynomials."""
 
-        N = len(cutoffs)
-        x, y = self._parse_modes_and_args(cutoffs)
+        N = self.num_modes
+        x = self.x.value * math.ones(N, dtype=self.x.value.dtype)
+        y = self.y.value * math.ones(N, dtype=self.y.value.dtype)
```

Comment:
Nice trick! Just leaving here a comment for future readers (this took me a while to understand): 

- this is taking scalars to vectors 
- if the arguments are vectors it will leave them untouched (well... multiplied by one)

hence this lines here take care of maintaining the API where one can use a single gate with 1) one parameter for multiple modes or 2) multiple parameters for multiple modes